pkg/operator/sync_openshiftcontrollermanager_v311_00.go

```diff
@@ -143,3 +160,78 @@ func manageOpenShiftControllerManagerPublicConfigMap_v311_00_to_latest(client co
 
 	return resourceapply.ApplyConfigMap(client, configMap)
 }
+
+func manageBuildAdditionalCAConfigMap(client coreclientv1.ConfigMapsGetter, configClient configclientv1.BuildsGetter, operatorConfig *v1alpha1.OpenShiftControllerManagerOperatorConfig) (string, bool, error) {
+	caMap := resourceread.ReadConfigMapV1OrDie(v311_00_assets.MustAsset("v3.11.0/openshift-controller-manager/build-additional-ca-cm.yaml"))
+	buildConfig, err := configClient.Builds().Get("cluster", metav1.GetOptions{})
+	if err != nil && !errors.IsNotFound(err) {
+		return "", false, err
+	}
+	if buildConfig == nil {
+		caMap.Data = nil
+		_, modified, err := resourceapply.ApplyConfigMap(client, caMap)
+		operatorConfig.Status.AdditionalTrustedCA = nil
+		return "", modified, err
```

Comment:
If no build CRD, clear the data in the controller manager's ConfigMap and nil the status for `AdditionalTrustedCA`